backend/crates/eiffelvis_core/src/app.rs

```diff
@@ -0,0 +1,35 @@
+use crate::graph_storage::ChunkedGraph;
+use crate::types::BaseEvent;
+
+use uuid::Uuid;
+
+type EiffelGraph = ChunkedGraph<Uuid, BaseEvent, String>;
+
+pub struct EiffelVisApp {
+    graph: EiffelGraph,
+}
+
+impl EiffelVisApp {
```

Comment:
I'm not entirely decided what direction I want to go with this, the vision is to make eiffelvis_http work as little as possible at least.

Reply:
So this type should probably be `Sync` so that the consumers dont have to lock mutexes / rwlocks all the time, instead just await function calls.